tests/integration/acp/dac/add_policy/with_empty_resource_test.go

```diff
@@ -16,25 +16,19 @@ import (
 	testUtils "github.com/sourcenetwork/defradb/tests/integration"
 )
 
-func TestACP_AddPolicy_OneResourceThatIsEmpty_Error(t *testing.T) {
+func TestACP_AddPolicy_OneResourceThatIsEmpty_DoesntError(t *testing.T) {
 	test := testUtils.TestCase{
 
 		Actions: []any{
 			testUtils.AddDACPolicy{
 				Identity: testUtils.ClientIdentity(1),
 
 				Policy: `
-                    name: a policy
-                    description: a policy
-
-                    actor:
-                      name: actor
-
-                    resources:
-                      users:
-                `,
-
-				ExpectedError: "BAD_INPUT",
+description: a policy
+name: a policy
+resources:
+- name: users
+`,
```

Comment:
question: this is because the `users` with have a default `expr` i.e. `owner`? In that case we should reflect this in or DPI/DRI.

suggestion: Add in addition another empty invalid test like before but with perhaps:

```suggestion
description: a policy
name: a policy
resources:
- name:
`,
```

Reply:
is this second test necessary? it would be a test testing acp, not defra